### PR TITLE
Fix #28 issue transparent/semi-transparent header on scroll

### LINKED ISSUE #28: Implement header style transparent/semi-transparent
Implement the same functionality as in the transparent/semi-transparent header on FOSSASIA.org.
* Ensure there are two logo versions, that change the color
* Ensure the font changes the color as needed

screenshot from 2017-11-30 14-20-21

### PR DESCRIPTION
**Pull request by kubami9**
Implemented transparent/semi-transparent navigation, just like in fossasia.org

### CODE REVIEW

**Comment by kubami9** [comment]
I attach 2 screenshots with preview. Original header background wasn't appearing on my machine, so I changed it to this dark blue just for testing and preview purpose. **This background is still the same, not touched :)**
main
secondary
*Second screenshot is made after scrolling down*

File: site/static/css/custom.css [hunk 1]
```diff
@@ -448,3 +448,23 @@ a:hover {
...
+.changed-nav {
+    background-color: rgba(255, 255, 255, 0.85) !important;
+}
+.changed-nav > div > .navbar-collapse > .nav > li> a,
+.changed-nav > div > .nav>li>a:focus{
+    color: #222 !important;
+}
+.navbar-collapse > .nav > li > a:active {
+    color: #5a5a5a !important;
+}
```
Comment (sayak119) [inline site/static/css/custom.css]: @kubami9 Can you add a newline here at the end of the file

File: site/static/js/onScrollMenu.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//Script for making menu on scroll as it is on fossasia.org
+$(window).scroll(function() {
+        if ($(window).scrollTop() > 1) {
+            $('.navbar-default').addClass('changed-nav');
+            $('#logo').replaceWith('<img id="logo" src="img/header_logo.svg" alt="Logo of the campaign" />')
+        } else {
+            $('.navbar-default').removeClass('changed-nav');
+            $('#logo').replaceWith('<img id="logo" src="img/logo.svg" alt="Logo of the campaign" />')
+        }
+    });
```
Comment (sayak119) [inline site/static/js/onScrollMenu.js]: Same a new line at he end of the file

File: site/static/css/custom.css (hunk 1 above)
Comment (sayak119) [inline site/static/css/custom.css]: @kubami9 Can you add a new line at the end of the file

File: site/static/js/onScrollMenu.js (hunk 2 above)
Comment (kubami9) [reply site/static/js/onScrollMenu.js]: Done ;)

**Comment by sayak119** [comment]
@kubami9 can u please merge your commits

**Comment by kubami9** [comment]
Squash, right?

**Comment by sayak119** [comment]
@kubami9 yup my bad :)

**Comment by kubami9** [comment]
Okay, please give me a second to get this stuff 😄

**Comment by kubami9** [comment]
How can I check if it's squashed?

**Comment by sayak119** [comment]
@kubami9 There will just 1 commit (In this case Fix #28. and Add new lines will become a single commit instead of 2)

**Comment by kubami9** [comment]
Ops. I did something wrong, now I have 4 commits :/

**Comment by sayak119** [comment]
@kubami9 Don't worry here's a guide https://help.github.com/articles/using-git-rebase-on-the-command-line/

**Comment by kubami9** [comment]
I did it! 😄 Thank you for this guide @sayak119

**Comment by kubami9** [comment]
So basically make logo white when on top and when scrolling the current one, right?

**Comment by kubami9** [comment]
Okay, now logo is white when the page is on top and same dark as current logo after scrolling down.

**Comment by kubami9** [comment]
I also squashed it all.

**Comment by anshumanv** [comment]
Good work! @kubami9 please wait for a maintainer of this project to be on, then he will review your PR and merge.

Meanwhile can you learn how to provide a deployment link so it eases out for us to review tasks and we can check and approve quickly. :smile: 
Cheers!

**Comment by kubami9** [comment]
Thank you! I'll do it as soon as I'll be back on my laptop! :)

**Comment by anshumanv** [comment]
Thanks for keeping a learning attitude. :smile:

**Comment by kubami9** [comment]
@anshumanv  I try to make deployment link, did I broke something now?

**Comment by kubami9** [comment]
It looks like I made preview link but with wrong directory https://kubami9.github.io/kuba-michalski-changes-publiccode.asia/
I can fix it but first I'd like to be sure that I didn't broke all my PR

**Comment by anshumanv** [comment]
Preview link can be made from the setting tab from your fork, no code changes needed. :smile: 
Please revert the recent changes you made.

**Comment by kubami9** [comment]
But this website is using Hugo framework and if I just make GH pages link preview it shows some raw text.
How can I revert my changes, a link would be very helpful

**Comment by anshumanv** [comment]
https://help.github.com/desktop/guides/contributing-to-projects/reverting-a-commit/
https://stackoverflow.com/questions/927358/how-to-undo-the-last-commits-in-git
Please see if this helps :smile:

**Comment by sayak119** [comment]
@kubami9 https://stackoverflow.com/questions/4114095/how-to-revert-git-repository-to-a-previous-commit can help

**Comment by anshumanv** [comment]
Then @kubami9 you can use http://surge.sh/ to deploy your code and provide a link using that, it's worth learning. :smile:

**Comment by kubami9** [comment]
Ufff... Thank you both! My heart stopped beating for a minute!

**Comment by anshumanv** [comment]
It's good that you encountered this error @kubami9, it taught you something while you handled it. :smile:

**Comment by kubami9** [comment]
I'm still struggling to learn this surge.sh and I'm also trying to track my phone since it's lost but to give the preview and get PR merged as fast as possible I give a link with few seconds video where I show changes, it's on Google Drive so you don't have to download it: https://drive.google.com/drive/folders/1Yql1V05FH74N8gUt6wVEr7dZvtR_0OTC?usp=sharing

**Comment by anshumanv** [comment]
Hey @kubami9 awesome screencast :smile:, if you're unable, it's okay this time, please learn and try to provide next time as it's a good practice to provide a link where users can test changes quickly and review.

Cheers!

**Comment by anshumanv** [comment]
@mariobehling please see and merge if everything's in place. Thanks :+1:

**Comment by mariobehling** [comment]
Ok, lets try this as work in progress. Thanks!

**Pull request merged**